packages/commonwealth/client/scripts/helpers/feature-flags.ts

```diff
@@ -7,14 +7,14 @@ import { InMemoryProvider } from '@openfeature/web-sdk';
 import { UnleashClient } from 'unleash-proxy-client';
 import { UnleashProvider } from '../../../shared/UnleashProvider';
 
-const buildFlag = (env: string) => {
+const buildFlag = (env: string, defaultValue: 'on' | 'off' = 'off') => {
   return {
     variants: {
       on: true,
       off: false,
     },
     disabled: false,
-    defaultVariant: env === 'true' ? 'on' : 'off',
+    defaultVariant: env === 'true' ? 'on' : defaultValue,
```

Comment:
I don't think we should be able to have it on by default. This extra flexibility can make the usage difficult to understand (We should always have the flag off unless specified as on)

Reply:
The defaultValue is 'off' if it's not set.  

It's just kind of a pain to shutdown my dev and then change the environment and restart to get new feature toggles.  

I don't feel strongly about it but this is a nice to have so I can quickly switch between the two.

I guess another hack is at the top of the script I could set process.env.FLAG_FOO to 'on' or something.